SqlScriptDom/Parser/TSql/TSql80ParserBaseInternal.cs

```diff
@@ -808,6 +808,15 @@ protected bool IsNextRuleBooleanParenthesis()
                         {
                             ++insideIIf;
                         }
+                        // if identifier is REGEXP_LIKE
+                        else if(NextTokenMatches(CodeGenerationSupporter.RegexpLike))
```

Comment:
Would it be more appropriate to add it to TSql170ParserBaseInternal or whichever version introduces this syntax?